sammo.py

```diff
@@ -3,18 +3,48 @@
 __contact__ = "info@hytech-imaging.fr"
 __copyright__ = "Copyright (c) 2021 Hytech Imaging"
 
-from .src.gui.session import SammoActionSession
+from .src.gui.session import SammoActionSession, ParentOfSammoActionSession
+from .src.gui.onOffEffort import SammoActionOnOffEffort, ParentOfSammoActionOnOffEffort
+from .src.core.session import SammoSession
 
 
-class Sammo:
+class Sammo(ParentOfSammoActionSession, ParentOfSammoActionOnOffEffort):
     def __init__(self, iface):
         self.iface = iface
-        self.toolBar = self.iface.addToolBar("Sammo ToolBar")
-        self.actionSession = SammoActionSession(self.iface.mainWindow(), self.toolBar)
+        self._toolBar = self.iface.addToolBar("Sammo ToolBar")
+        self._session = SammoSession()
+        self.actionSession = SammoActionSession(self)
+        self.actionOnOffSession = SammoActionOnOffEffort(self)
 
     def initGui(self):
         self.actionSession.initGui()
+        self.actionOnOffSession.initGui()
```

Comment:
I propose to hide the `initGui` call directly in the  `SammoActionOnOffEffort` cst.